app/views/DR/UserInfoScreen/index.js

```diff
@@ -219,18 +250,18 @@ export default function UserInfo({ navigation }) {
                   ]}
                   onPress={async () => {
                     //Send data to API
-
-                    isLoading = true;
-                    if (await sendDataToApi()) {
-                      console.log('onpress');
-                      isLoading = false;
-                      getAge(birth);
-                      navigation.navigate('Report');
-                      closeDialog(false);
+                    let response = await sendDataToApi();
+                    if (response.valid != undefined) {
```

Comment:
if comparing specifically for undefined use !== instead of !=

Reply:
Done